test/ng/locationSpec.js

```diff
@@ -57,13 +60,14 @@ describe('$location', function() {
         search: '',
         setAttribute: angular.noop
       };
-    }));
+    });
 
-    afterEach(inject(function($sniffer) {
-      if (msie) return;
+    afterEach(function() {
+      // Support: non-Windows browsers
+      if (msie || /\bEdge\/[\d\.]+\b/.test(window.navigator.userAgent)) return;
```

Comment:
I guess it doesn't hurt when it's used just in tests... (I would definitely *not* want sth like that in source). But would you like to have it in tests globally or just in `locationSpec`? If globally then we should define similar variables for other browsers.

I'm a little afraid of making it too easy to use UA-sniffing, though.